packages/monorepo-scripts/src/publish.ts

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env node
+
+import * as fs from 'fs';
+import lernaGetPackages = require('lerna-get-packages');
+import * as _ from 'lodash';
+import * as moment from 'moment';
+import * as path from 'path';
+import { exec as execAsync, spawn } from 'promisify-child-process';
+import semverDiff = require('semver-diff');
+import semverSort = require('semver-sort');
+
+import { constants } from './constants';
+import { Changelog, Changes, SemVerIndex, UpdatedPackage } from './types';
+import { utils } from './utils';
+
+const IS_DRY_RUN = process.env.IS_DRY_RUN === 'false';
```

Comment:
I find this a bit confusing, when dry run then it should be a "test" run with no dangerous side effects.

```
const IS_DRY_RUN = process.env.IS_DRY_RUN === 'true'
```

and in the package.json

```
"script:publish:dry": "IS_DRY_RUN=true yarn script:publish"
```

Setting DRY_RUN to false feels like it shouldn't perform a dry run, and therefor perform the dangerous side effects.

Reply:
Yep, agreed. Flipped it.